packages/env-loader/src/index.ts

```diff
@@ -33,8 +33,8 @@ export const processEnv = async (command: string): Promise<void> => {
   process.env = {
     ...process.env,
     ...{
-      ...envVars,
```

Comment:
This ordering is switched to allow developer override of feature flags via their env vars, either locally or on a given environment. In the previous setup, the only way to override the feature flags was via cli options, and that will be difficult for devs to work with and nearly impossible to utilize on say a dev or staging environment.